Dockerfile

```diff
@@ -1,8 +1,8 @@
 # syntax = docker/dockerfile:experimental
 
-ARG KUBECTL_VERSION=1.29.4
-ARG HELM_VERSION=3.14.4
-ARG KUSTOMIZE_VERSION=5.4.1
+ARG KUBECTL_VERSION=1.29.7
+ARG HELM_VERSION=3.15.3
```

Comment:
Just wanted to raise concern that this Helm version support kubernetes 1.30 to 1.27 which differs from what[ we support ](https://www.okteto.com/docs/release-notes/#1220). I don't think we should have any problem but leaving it here in case we face a problem